schemas/event.yml

```diff
@@ -277,6 +277,14 @@
             - end
             - info
             - start
+        - name: registry
+          description: >
+            Use this category of event for events related to the Windows registry.
+          expected_event_types:
+            - access
```

Comment:
I don't remember the distinction between access/read, but there is at least one example that comes to mind. Retrieving the permissions or contents of the SAM key is helpful telemetry for endpoint telemetry, so I think this one makes sense, too.

Reply:
There's actually no `event.type` value "read". There's only "access". So no distinction to make, just use "access" :-)

Reply:
@rw-access Although to get back to my initial question, I guess I was missing the word "even": is it *even* possible to detect registry key access?

If the registry doesn't offer the ability to be notified of a read or access event, we could remove the value "access".

Reply:
there is an ability to detect registry reads, so I think we can keep **access**.

Reply:
Perfect, thanks for confirming 🙇